packages/plugins/codelens/extension.ts

```diff
@@ -0,0 +1,13 @@
+import SQLTools from '@sqltools/core/plugin-api';
+import { languages } from 'vscode';
+import SQLToolsCodeLensProvider from './provider';
+
+export default class CodeLensPlugin implements SQLTools.ExtensionPlugin {
+  register(extension: SQLTools.ExtensionInterface) {
+    const codeLensDisposable = languages.registerCodeLensProvider(
+      { language: 'sql' },
```

Comment:
Could "language" be configurable?

Reply:
Yes, the plan is to have it in settings as well.

This is the very beginning, I wanted to test how it would be.